### PR TITLE
[GridBundle] CS fixes

### PR DESCRIPTION
**Pull request by aRn0D**
| Q             | A
| ------------- | ---
| Bug fix?      | CS fixes
| New feature?  | no 
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | no
| License       | MIT
| Doc PR        | -

### CODE REVIEW

File: src/Sylius/Bundle/GridBundle/DataSource/Doctrine/ORM/DataSource.php [hunk 1]
```diff
@@ -36,9 +36,11 @@ class DataSource implements DataSourceInterface
     private $expressionBuilder;
 
     /**
-     * @param EntityRepository
+     * @param EntityRepository $repository
+     * @param null             $method
```
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/GridBundle/DataSource/Doctrine/ORM/DataSource.php]: I suppose ``` @param string|null $method ```

File: src/Sylius/Bundle/GridBundle/Twig/GridExtension.php [hunk 2]
```diff
@@ -137,10 +144,16 @@ public function renderValue(\Twig_Environment $twig, $object, GridView $gridView
      * @param GridView          $gridView
      * @param Action            $actionDefinition
      * @param Request           $request
-     * @param $object
+     * @param null              $object
```
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/GridBundle/Twig/GridExtension.php]: ``` @param mixed|null $object ``` ?

**Comment by aRn0D** [comment]
@michalmarcinkowski done

**Comment by michalmarcinkowski** [comment]
:+1:

**Comment by michalmarcinkowski** [comment]
Thanks Arnaud! This will be included :)

**Pull request closed without merging**